### PR TITLE
Formatting cleanup

### PR DESCRIPTION
**Pull request by har00ga**
Adjusted formatting on the getting started guide and removed outdated sections. It seems the FIL faucet that was linked is no longer active so I've removed that as well; if there is a new service that could be listed instead, happy to readd it.

### CODE REVIEW

**Comment by har00ga** [comment]
A quick note before I head off, there's still 1 more thing that needs to be adjusted:

```
export WALLET_ADDR=`go-filecoin address ls`    # fetch your wallet address into a handy variable
MESSAGE_CID=`curl -X POST -F "target=${WALLET_ADDR}" "#" | cut -d" " -f4`
```
`"#"` was formerly the query address of the kittyhawk website. I tried a couple times tonight to get a similar link from faucet.testnet.filecoin.io but unfortunately seems the service is temporarily down (I imagine from the heavy load today). Hopefully it'll be back up in the morning and I will add it in then. :slightly_smiling_face:

File: go-filecoin-tutorial/Getting-Started.md [hunk 1]
```diff
@@ -32,46 +29,46 @@ git clone https://github.com/filecoin-project/go-filecoin.git /path/to/filecoin-
...
 
-Due to our use of `cgo`, you'll need a C compiler to build go-filecoin whether you're using a prebuilt library or it yourself from source. 
-If you want to use `gcc` (e.g. `export CC=gcc`) when building go-filecoin, you will need to use v7.4.0 or higher.
+Due to the use of `cgo` in `go-filecoin`, a C compiler is required to build it whether a prebuilt library is being used or it is compiled from source. To use `gcc` (e.g. `export CC=gcc`), v7.4.0 or higher is required.
 
 The build process will download a static library containing the [Filecoin proofs implementation](https://github.com/filecoin-project/rust-fil-proofs) (which is written in Rust).
 
-> If instead you wish to build proofs from source, you'll need (1) Rust development environment and (2) to set the environment variable `FFI_BUILD_FROM_SOURCE=1`.
-More info at [filecoin-ffi](https://github.com/filecoin-project/filecoin-ffi).
+> **NOTICE:** To build proofs from source,  (1) a Rust development environment must be installed and (2) The environment variable `FFI_BUILD_FROM_SOURCE=1` must be set. More information can be found in [filecoin-ffi](https://github.com/filecoin-project/filecoin-ffi).
```
Comment (mishmosh) [inline go-filecoin-tutorial/Getting-Started.md:40]:
```suggestion
> **NOTICE:** To build proofs from source, (1) a Rust development environment must be installed and (2) the environment variable `FFI_BUILD_FROM_SOURCE=1` must be set. More information can be found in [filecoin-ffi](https://github.com/filecoin-project/filecoin-ffi).
```

File: go-filecoin-tutorial/Getting-Started.md [hunk 2]
```diff
@@ -32,46 +29,46 @@ git clone https://github.com/filecoin-project/go-filecoin.git /path/to/filecoin-
...
 
 #### Installing dependencies
 
-First, load all the Git submodules.
+1. Load all the Git submodules:
```
Comment (mishmosh) [inline go-filecoin-tutorial/Getting-Started.md:44]: Nice!

File: go-filecoin-tutorial/Getting-Started.md [hunk 3]
```diff
@@ -131,86 +128,68 @@ go-filecoin swarm connect <any-filecoin-node-mulitaddr>
     
  > **NOTICE:** This can be **slow** the first time. The filecoin node needs a large parameter file for proofs, stored in `/tmp/filecoin-proof-parameters`. It's usually generated by the `deps` build step. If these files are missing they will be regenerated, which can take up to an hour. We are working on a better solution.
 
-Your node should now be connected to some peers and will begin downloading and validating the blockchain.
-
-6. Check your connectivity:
+6. Check the nodes connectivity:
```
Comment (mishmosh) [inline go-filecoin-tutorial/Getting-Started.md:131]:
```suggestion
6. Check the node's connectivity:
```

File: go-filecoin-tutorial/Getting-Started.md [hunk 4]
```diff
@@ -131,86 +128,68 @@ go-filecoin swarm connect <any-filecoin-node-mulitaddr>
...
         ```
-1. The faucet will provide you with a message CID. If your chain is already synced with the network, this message should be processed in about 30 seconds. You can run the following command to wait for confirmation:
+3. The faucet will provide a message CID. If thr chain is already synced with the network, this message should be processed in about 30 seconds. The following command can be run in order to wait for confirmation:
 
     ```sh
     go-filecoin message wait ${MESSAGE_CID}
     ```
 
-1. To verify that the FIL has landed in your wallet, check your wallet balance:
+4. To verify that the FIL has landed in the wallet by checking the wallet balance:
```
Comment (mishmosh) [inline go-filecoin-tutorial/Getting-Started.md:176]:
```suggestion
4. Verify that the FIL has landed in the wallet by checking the wallet balance:
```

**Pull request merged**